defaults/templates/the-build/build.acquia.properties.yml

```diff
@@ -1,6 +1,10 @@
 # @file
 # Configuration properties specific to the Acquia deployment.
-build:
-  drupal:
-    settings: .the-build/drupal.settings.build-acquia.php
-    services_dest: artifacts/dev/null
+drupal:
+  twig:
+    debug: false
+  sites:
+    default:
+      build:
+        settings: .the-build/drupal.settings.build-acquia.php
```

Comment:
I think this needs to be `settings_template`.  I was building an artifact for Acquia, and it was using the local build settings template.

Reply:
nice catch!